php/WP_CLI/Dispatcher/Subcommand.php

```diff
@@ -174,57 +178,63 @@ private function prompt_args( $args, $assoc_args ) {
 
 				$repeat = false;
 				do {
-					if ( ! $repeat )
+					if ( ! $repeat ) {
 						$key_prompt = $current_prompt . $key_token;
-					else
-						$key_prompt = str_repeat( " ", strlen( $current_prompt ) ) . $key_token;
+					} else {
+						$key_prompt = str_repeat( ' ', strlen( $current_prompt ) ) . $key_token;
+					}
 
 					$key = $this->prompt( $key_prompt, $default );
-					if ( false === $key )
+					if ( false === $key ) {
 						return array( $args, $assoc_args );
+					}
 
 					if ( $key ) {
 						$key_prompt_count = strlen( $key_prompt ) - strlen( $value_token ) - 1;
-						$value_prompt = str_repeat( " ", $key_prompt_count ) . '=' . $value_token;
+						$value_prompt = str_repeat( ' ', $key_prompt_count ) . '=' . $value_token;
 
 						$value = $this->prompt( $value_prompt, $default );
-						if ( false === $value )
+						if ( false === $value ) {
 							return array( $args, $assoc_args );
+						}
 
-						$assoc_args[$key] = $value;
+						$assoc_args[ $key ] = $value;
 
 						$repeat = true;
 					} else {
 						$repeat = false;
 					}
-
-				} while( $repeat );
+				} while ( $repeat );
 
 			} else {
 
 				$prompt = $current_prompt . $spec_arg['token'];
-				if ( 'flag' == $spec_arg['type'] )
+				if ( 'flag' == $spec_arg['type'] ) {
 					$prompt .= ' (Y/n)';
+				}
 
 				$response = $this->prompt( $prompt, $default );
-				if ( false === $response )
+				if ( false === $response ) {
 					return array( $args, $assoc_args );
+				}
 
 				if ( $response ) {
 					switch ( $spec_arg['type'] ) {
 						case 'positional':
-							if ( $spec_arg['repeating'] )
+							if ( $spec_arg['repeating'] ) {
 								$response = explode( ' ', $response );
-							else
+							} else {
 								$response = array( $response );
+							}
 							$args = array_merge( $args, $response );
 							break;
 						case 'assoc':
-							$assoc_args[$spec_arg['name']] = $response;
+							$assoc_args[ $spec_arg['name'] ] = $response;
 							break;
 						case 'flag':
-							if ( 'Y' == strtoupper( $response ) )
-								$assoc_args[$spec_arg['name']] = true;
+							if ( 'Y' == strtoupper( $response ) ) {
```

Comment:
Also not addressed, would be obvious changes of comparison from equality to identity, like this.